src/js/webrequest.js

```diff
@@ -172,6 +172,23 @@ function onBeforeSendHeaders(details) {
     } else {
       return {};
     }
+
+  } else if (badger.isSpoofReferrerEnabled()) {
+    if (badger.isPrivacyBadgerEnabled(tabDomain) && badger.isPrivacyBadgerEnabled(requestDomain)) {
+      // spoof referer header for third party requests
+      const refererHeader = details.requestHeaders.find(header => header.name === "Referer");
```

Comment:
Style nit: `refer_header` (lowercase with underscores for for non-object/non-object-property variables).